LocalSettings.php

```diff
@@ -3146,8 +3147,12 @@
 			),
 		),
 		'+wikicanadawiki' => array(
-			'Founder' => array(
+			'founder' => array(
 				'read' => true,
+				'checkuser' => true,
```

Comment:
Why? Is there some sort of technical/database restrictions that prevent non-Stewards from using theses permissions?